### PR TITLE
Add aarch64 linux build

### PR DESCRIPTION
**Pull request by mlund**
This adds support for arm-based linux distributions.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mlund** [comment]
@conda-forge-admin, please rerender

**Comment by mattwthompson** [comment]
Should there be ARM builds in the matrix on Azure? I'm not seeing any new ones: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=866942&view=logs&jobId=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e

File: conda-forge.yml [hunk 1]
```diff
@@ -9,3 +9,4 @@ github:
 provider:
   win: azure
 test_on_native_only: true
+provider: {linux_aarch64: default}
```
Comment (marscher) [inline conda-forge.yml]: are you sure this is the right syntax?

**Comment by mlund** [comment]
> Should there be ARM builds in the matrix on Azure? I'm not seeing any new ones: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=866942&view=logs&jobId=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e

Hmm, not sure - I don't see the macOS builds there either, but perhaps they do not run on Azure. It might be possible to alternatively use `provider: {linux_aarch64: azure}`(?). I followed the information found here: https://thepracticalsysadmin.com/building-multiarch-conda-forge-recipes/

File: conda-forge.yml (hunk 1 above)
Comment (marscher) [reply conda-forge.yml]: https://conda-forge.org/docs/maintainer/conda_forge_yml.html#linux-aarch64

**Comment by marscher** [comment]
> > Should there be ARM builds in the matrix on Azure? I'm not seeing any new ones: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=866942&view=logs&jobId=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e
> 
> Hmm, not sure - I don't see the macOS builds there either, but perhaps they do not run on Azure. It might be possible to alternatively use `provider: {linux_aarch64: azure}`(?). I followed the information found here: https://thepracticalsysadmin.com/building-multiarch-conda-forge-recipes/

I think this is outdated. Please have a looked at the previously posted documentation.

File: conda-forge.yml (hunk 1 above)
Comment (mlund) [reply conda-forge.yml]: I followed the instructions from [here](https://thepracticalsysadmin.com/building-multiarch-conda-forge-recipes/). In e.g. the [Faunus feedstock](https://github.com/conda-forge/faunus-feedstock/blob/main/conda-forge.yml) `default` is however `azure`. Nonetheless, aarch64 builds are triggered under Checks. No expert here, just trying to solve the problem that I cannot use `mdtraj` in aarch64-linux Docker images on apple silicon.

**Comment by marscher** [comment]
@conda-forge-admin please rerender

**Comment by mlund** [comment]
OK, I changed to `linux_aarch64: {enabled: True}`, but now to not see any arm builds.

**Comment by marscher** [comment]
try to ping somebody in gitter.im conda-forge channel

**Comment by mlund** [comment]
> try to ping somebody in gitter.im conda-forge channel


Thanks, but I'm not sure what that means? Btw. the manual states that `linux_aarch64: {enabled: True}` is _largely an internal value_. Reading further, I think my original proposal was possibly OK, see here: https://conda-forge.org/docs/maintainer/conda_forge_yml.html#provider

**Comment by mattwthompson** [comment]
@mlund https://app.gitter.im/#/room/#conda-forge:matrix.org

I'm pretty sure the way to do these migrations is let the bot try (https://github.com/conda-forge/conda-forge-pinning-feedstock/pulls?q=is%3Apr+aarch64), or at least that's what I did the last time I added a platform to an existing feedstock

**Comment by mlund** [comment]
> @mlund https://app.gitter.im/#/room/#conda-forge:matrix.org
> 
> I'm pretty sure the way to do these migrations is let the bot try (https://github.com/conda-forge/conda-forge-pinning-feedstock/pulls?q=is%3Apr+aarch64), or at least that's what I did the last time I added a platform to an existing feedstock

Thanks, I think I got it and opened this: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5454

**Pull request closed without merging**